microservices/device-simulation-dotnet/deployment/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "sim-deployment",
+  "version": "1.0.0",
+  "description": "Deployment script for Azure IoT Device Simulation",
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./dist/start.js",
```

Comment:
I'm surprised that main (on line 5) doesn't match line 7 (start.js).  Is this intentional?

Reply:
These files have been removed.